packages/manager/modules/pci/src/components/project/billing/resource-usage/controller.js

```diff
@@ -3,4 +3,14 @@ export default class BillingResourceUsageCtrl {
   constructor(CucRegionService) {
     this.CucRegionService = CucRegionService;
   }
+
+  static hourlyQuantity(quantity) {
+    switch (quantity.unit) {
```

Comment:
**question:** Does the value can be returned by the API instead?

Reply:
I discussed with @christophe-rannou and for a short term period, that's the only patch that can be applied.
So to improve readability we will integrate this patch.